cw-orch-daemon/src/tx_resp.rs

```diff
@@ -124,7 +126,22 @@ impl From<TxResponse> for CosmTxResponse {
             gas_wanted: tx.gas_wanted as u64,
             gas_used: tx.gas_used as u64,
             timestamp: parse_timestamp(tx.timestamp).unwrap(),
-            events: tx.events,
+            events: tx.events.into_iter().map(|e| {
+                println!("e: {:#?}", e);
+                Event {
+                    r#type: e.r#type,
+                    attributes: e
+                        .attributes
+                        .into_iter()
+                        .map(|a| EventAttribute {
+                            key: String::from_utf8(a.key.clone().to_vec()).unwrap(),
+                            value: String::from_utf8(a.value.clone().to_vec()).unwrap(),
+                            index: false,
+                        })
+                        .collect(),
+                }
+            }
+            ).collect(),
```

Comment:
@Kayanski not sure what we want to do in this case. The expected `Event` is from v0.38.

Reply:
You fixed that now didn't you ?